md-dao/src/main/java/com/wipro/ats/bdre/md/dao/HistoryDataImportDAO.java

```diff
@@ -41,8 +41,15 @@
     private static final Logger LOGGER = Logger.getLogger(HistoryDataImportDAO.class);
     @Autowired
     SessionFactory sessionFactory;
-
-    public List<com.wipro.ats.bdre.md.beans.table.Process> historyDataImport(IntermediateInfo intermediateInfo) throws Exception {
+    private static final String _IMPORT="_import";
+    private static final String IMPCOMMON="imp-common";
+    private static final String DATAIMPORTPROPERTIES="properties for  data import";
+    private static final String RAW_TABLE="raw-table";
+    private static final String TABLE_NAME="table_name";
+    private static final String RAW_TABLE_NAME="Raw Table Name";
+    private static final String TABLE_DB="table_db";
+    private static final String BASE_TABLE="base-table";
+    public List<com.wipro.ats.bdre.md.beans.table.Process> historyDataImport(IntermediateInfo intermediateInfo){
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) The Cyclomatic Complexity of this method "historyDataImport" is 23 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)